app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -116,6 +124,7 @@ class HomeActivityTest {
 
   private val internalProfileId: Int = 0
   private val internalProfileId1: Int = 1
+  private val longNameInternalProfileId: Int = 3
```

Comment:
Why do we need a different profile Id? I am guessing other than Admin, which use `internalProfileId ` all other profile can have long names. So, I think we can use `internalProfileId1 `.

Reply:
Also, when we do `initializeProfiles` 0 is Admin, 1 is Ben, 2 is Nikita. 

But, when I saw `addLongNameProfile` function, we are login using admin profile. `ProfileId.newBuilder().setInternalId(0).build()`

Reply:
I am adding a review below to throw some light on this.

Reply:
FIxed the mistake